src/status_im/contexts/wallet/save_address/events.cljs

```diff
@@ -1,4 +1,4 @@
-(ns status-im.contexts.wallet.events.saved-addresses
+(ns status-im.contexts.wallet.save-address.events
```

Comment:
@shivekkhurana - 
I believe we had been going with an events file living close to its use case folder, rather than a folder of events with files for each functionality.

I moved this save-address stuff to here as I think it can be a bit more generic than within send folder too.
The flows for save address are actually used in the wallet settings also so maybe it makes sense to have it's own folder.
wdyt? 🤔